hydra-access-controls/lib/hydra-access-controls.rb

```diff
@@ -2,6 +2,7 @@
 require 'active-fedora'
 require 'blacklight'
 require 'cancan'
+require 'action_view' # https://github.com/haml/haml/issues/695
```

Comment:
I'm thinking maybe `active-fedora/lib/active_fedora/railtie.rb` is the place to put this.  It doesn't really belong in rdf-rdfa, because the bug only occurs when `Rails` is defined.